.circleci/config.yml

```diff
@@ -0,0 +1,56 @@
+version: 2
+
+workflows:
+  version: 2
+  install-airflow-chart:
+    jobs:
+      - 'airflow-on-k8s-14'
+      - 'airflow-on-k8s-15'
+      # - 'airflow-on-k8s-16'
+      # - 'airflow-on-k8s-17'
```

Comment:
Future PRs will come to enable support for newest kube versions. These are not working at the moment.